.github/workflows/mega-linter-for-runner.yml

```diff
@@ -125,7 +125,8 @@ jobs:
             (
               github.event_name == 'push' ||
               github.event.pull_request.head.repo.full_name == github.repository
-            )
+            ) &&
+            !contains(github.event.head_commit.message, 'skip fix')
```

Comment:
Already in https://github.com/oxsecurity/megalinter/pull/3032/files#diff-f339c840e1de23f5dc489a9282db9dff6a8bcade192e4f4d7edb239548320552R285-R286